src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -354,13 +359,18 @@ class HoldRequest extends React.Component {
 
     const searchUrl = basicQuery(this.props)({});
 
+    if (this.isLoading()) {
```

Comment:
Why not put this up higher? There is a lot of code that is processed for no reason before getting to this point.

Also, something like this would give the page more height:
```
<Fragment>
          <LoadingLayer
            status={this.isLoading()}
            title="Requesting"
          />
          <div>
            <div className="nypl-request-page-header">
              <div className="nypl-full-width-wrapper">
                <div className="row">
                  <div className="nypl-column-full">
                    <Breadcrumbs
                      type="hold"
                    />
                    <h1 id="item-title" tabIndex="0" id="mainContent">Item Request</h1>
                  </div>
                </div>
              </div>
            </div>

            <div className="nypl-full-width-wrapper">
              <div className="row">
                <div className="nypl-column-three-quarters" />
              </div>
            </div>
          </div>
        </Fragment>
```

Reply:
👍 Added this and moved it up to the top of the render method.

Reply:
I'm not a fan of this approach because now the same skeleton HTML is duplicated twice for the page.
It's already too late since this is merged, but I'd add a ticket to refactor this later.

Reply:
Yeah. I think this component could use an overhaul. I'm imagining the logic being moved into a function instead of all being spread out in the render method.